.github/workflows/build.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - dasharo
     tags:
-      - 'dasharo_qemu_v*'
+      - 'qemu_q35_v*'
```

Comment:
@macpijan since you created edk2 CI, it would be great if you could review the changes in this file.

Reply:
@pietrushnic I'm fine with these